core/psh/ps/ps.c

```diff
@@ -42,40 +42,62 @@ static int psh_ps_cmpcpu(const void *t1, const void *t2)
 }
 
 
-void psh_psinfo(void)
+static void psh_psinfo(void)
 {
 	printf("prints processes and threads");
 }
 
 
-int psh_ps(int argc, char **argv)
+static void usage(const char *progname)
+{
+	printf("Usage: %s [options]\n", progname);
+	printf("\nDisplaying:\n");
+	printf("    -t    Show threads\n");
+	printf("    -f    Show full commandline\n");
+	printf("    -h    Show help instead\n");
+	printf("\nSorting:\n");
+	printf("    -c    Sort by current CPU usage [default]\n");
+	printf("    -n    Sort by name\n");
+	printf("    -p    Sort by PID\n");
+}
+
```

Comment:
Extra new line missing between 2 function blocks.

Reply:
thanks, I thought that "two newlines" rule apply to function declarations only. Fixed.